posts/2024-07-26-actuated-for-gitlab.md

```diff
@@ -0,0 +1,217 @@
+---
+title: Secure CI for GitLab with Actuated.
+description: Learn how Actuated can improve your GitLab CI.
+tags:
+  - gitlab
+author_img: welteki
+date: "2024-07-26"
+---
+
+Last year we introduced a [tech preview for Actuated for GitLab CI](/blog/secure-microvm-ci-gitlab). Actuated was created to offer low maintenance, secure self-hosted CI runners that make optimal use of your available compute resources.
```

Comment:
'Actuated was created to offer low maintenance, secure self-hosted CI runners that make optimal use of your available compute resources.'=>

`Actuated reduces management overheads of self-hosted runners and provides a secure, ephemeral microVM for every job.`